Traceback (most recent call last):
  File "/home/mjirka/anaconda3/lib/python3.9/site-packages/nbclient/client.py", line 730, in _async_poll_for_reply
    msg = await ensure_async(self.kc.shell_channel.get_msg(timeout=new_timeout))
  File "/home/mjirka/anaconda3/lib/python3.9/site-packages/nbclient/util.py", line 96, in ensure_async
    result = await obj
  File "/home/mjirka/anaconda3/lib/python3.9/site-packages/jupyter_client/channels.py", line 230, in get_msg
    raise Empty
_queue.Empty

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mjirka/anaconda3/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/home/mjirka/anaconda3/lib/python3.9/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/mjirka/anaconda3/lib/python3.9/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/mjirka/anaconda3/lib/python3.9/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/home/mjirka/anaconda3/lib/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/home/mjirka/anaconda3/lib/python3.9/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/home/mjirka/anaconda3/lib/python3.9/site-packages/nbclient/client.py", line 949, in async_execute_cell
    exec_reply = await self.task_poll_for_reply
  File "/home/mjirka/anaconda3/lib/python3.9/site-packages/nbclient/client.py", line 754, in _async_poll_for_reply
    await self._async_handle_timeout(timeout, cell)
  File "/home/mjirka/anaconda3/lib/python3.9/site-packages/nbclient/client.py", line 801, in _async_handle_timeout
    raise CellTimeoutError.error_from_timeout_and_cell(
nbclient.exceptions.CellTimeoutError: A cell timed out while it was being executed, after 30 seconds.
The message was: Cell execution timed out.
Here is a preview of the cell contents:
-------------------
['#', '# priklad minimalizace funkce jedne promenne metodou zlateho rezu', '', '# zadejme si funkci -sin(x)', 'def f(x):']
...
['    # pokud je minimum je v intervalu < b,c > (ten navic obsahuje bod d),        ', '    # bod d se stane novym bodem b', '    # a levy okraj presuneme do puvodniho bodu b    ', '', "print('Minimum je v intervalu <',a,',',c,'>')"]
-------------------

